dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -731,7 +751,8 @@ public void ContentFallbackTrue_PageFallbackFalse_PageEnglish_ViewEnglishContent
      * @throws Exception
      */
     @Test
-    public void constantField_notUpdatedCache_whenChanged() throws Exception{
+    @UseDataProvider("cases")
+    public void constantField_notUpdatedCache_whenChanged(final Container container, final Template template) throws Exception{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5041322)